### ISSUE TITLE
Youtube-DLG 0.4 Downloads All Videos Successfully But Status Shows Error

### ISSUE THREAD

**Issue by Jaladhjin** [MrS0m30n3/youtube-dl-gui#533]
I'm using Youtube-DLG 0.4 on Windows & am able to download YT videos successfully but it's the strangest thing after I'm notified Downloads have completed the Status all show error on every video.

But it all worked the videos downloaded successfully.

Individual videos not from a playlist or I believe even from a playlist.

But the program totally works & the video downloads successfully.

Is this a known issue?

**Comment by axcore** [comment]
>Is this a known issue?

Yes, you need to download with yt-dlp instead of youtube-dl. This project is abandoned, so use an active fork like [this one](https://github.com/axcore/tartube) or [this one](https://github.com/oleksis/youtube-dl-gui).\
